src/am_map.c

```diff
@@ -247,7 +247,7 @@ int automap_grid = 0;
 
 boolean automapactive = false;
 
-boolean automapoverlay = false;
+int automapoverlay = false;
```

Comment:
Maybe named constants like this would be helpful instead of literals:
```c
typedef enum
{
  overlay_off,
  overlay_on,
  overlay_dark,
} overlay_t;

overlay_t automap_overlay;
```

Reply:
You could reuse this for `menu_background` as well I guess :^)

Reply:
`menu_background = 0` means that the background is **on**, though. We can't exactly reuse this enum, but we could very well make one for it.

Reply:
Yes, I agree that an `enum` would make the code easier to read. The menu background would need a slightly differently named `enum`, but that's stuff for another PR.